### DISCUSSION TITLE
Support multiple webdrivers for E-Mail reports

### DISCUSSION THREAD

**Discussion by EinavDanielDX** [apache/superset#24086] [Ideas]
At the moment, when sending an email report from Chrome, the webdriver installed on superset container/server MUST be consistent with the clients chrome version.
This is not practical since different users of the same superset server might have different browser versions. furthermore - it requires the IT to keep prompting the webdriver on the server for any new Chrome version.

I'd suggest using python `webdriver-manager` to support multiple driver versions - thus, multiple chrome versions.
See reference [here](https://pypi.org/project/webdriver-manager/).

**Comment by rusackas** [comment]
Hmm... I'm going to close this since it's quite stale, and more feature request / PR suggestion than a bug report. 

However, it sure sounds like an interesting idea. I'll move this to a Github Discussion and ping a few people there that might be able to support it and/or review a PR if you're interested in contributing one.
